src/gettingstarted/local-development/build-locally.md

```diff
@@ -0,0 +1,47 @@
+# Local development
+
+## Build site locally
+
+Now that you've opened tunnels into your services, you'll have access to all of your data in your environment. All that's left now is to actually build the site.
+
+<asciinema-player src="/videos/asciinema/build.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+1. **Build the site**
+
+    From the repository root, run the command
+
+    ```bash
+    platform build
+    ```
+
+    The Platform CLI will first ask you for the source directory and the build destination, then it will use your `.platform.app.yaml` file to execute the build process locally.
+
+2. **Verify**
+
+    Move to the build destination (i.e. `cd _www`) and then run a local web server to verify the build.
+
+    {% codetabs name="PHP", type="php" -%}
+    php -d variables_order=EGPCS -S localhost:8001
+    {%- language name="Python", type="py" -%}
+    python3 -m http.server 8000
+    {%- endcodetabs %}
```

Comment:
Is it worth noting that for Node and Go you can just execute the program directly and it does its own listening?